code-submissions-clustering-core/src/main/kotlin/org/jetbrains/research/code/submissions/clustering/model/Language.kt

```diff
@@ -1,12 +1,10 @@
 package org.jetbrains.research.code.submissions.clustering.model
 
-import com.jetbrains.python.PythonFileType
-
 /**
  * Enum class for possible code submissions' languages.
  * @property extension corresponding file extension
  */
 enum class Language(val extension: String) {
-    PYTHON(PythonFileType.INSTANCE.defaultExtension),
+    PYTHON("py"),
```

Comment:
Why Šid you change it?

Reply:
I changed it because PythonFileType requires a Python Plugin dependency. On the one hand, I moved the rest of such cases to code-submissions-clustering-ij, where it is available, but it seems not worth removing supported languages from models in core module.